tsfresh/feature_selection/feature_selector.py

```diff
@@ -118,8 +119,10 @@ def check_fs_sig_bh(X, y, settings=None):
     # Helper function which wraps the _calculate_p_value with many arguments already set
     f = partial(_calculate_p_value, y=y, settings=settings, target_is_binary=target_is_binary)
 
+    chunksize = helper_functions.calculate_best_chunksize(X, settings)
```

Comment:
Is the length of X the same as the length of the features list? Anyhow, I think it is easier readable to have the features list here directly